### PR TITLE
feat: retain user selected skin colour after user reacts with the selected skin colour

### PR DESCRIPTION
**Pull request by arjita-mitra**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - retain user selected skin colour

- The **PR Description**
  - when a user change the skin colour of emoji and react to a emoji with that skin colour then that skin colour should be retained when the emoji picker re-opened.
----

### CODE REVIEW

**Review by V-Gira — APPROVED** [review]

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx [hunk 1]
```diff
@@ -92,6 +94,12 @@ const EmojiPickerContainer: FC<EmojiPickerContainerProps> = ({
       resetActionMenuStates();
     }
   }
+
+  function getSkinTone() {
+    const currentSkinTone = localStorage.getItem('activeSkinTone');
+    const skinTone = currentSkinTone ? (currentSkinTone as SkinTones) : SkinTones.NEUTRAL;
+    return skinTone;
```
Comment (arjita-mitra) [inline src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx:101]: mine is more readable

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx [hunk 2]
```diff
@@ -92,6 +94,12 @@ const EmojiPickerContainer: FC<EmojiPickerContainerProps> = ({
...
+  }
```
Comment (przemvs) [inline src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx:102]:
```suggestion
  function getSkinTone() {
    const currentSkinTone: SkinTones | null = localStorage.getItem('activeSkinTone');
    return currentSkinTone || SkinTones.NEUTRAL;
  }
  
```

Comment (przemvs) [reply src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx:102]: You can also do smth like that:
`function getSkinTone(): SkinTones {`

Comment (arjita-mitra) [reply src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx:102]: its not required to make this change. mine is perfectly readable and working so i would keep it that way

**Pull request merged**